lib/enigma.rb

```diff
@@ -1,9 +1,22 @@
 require 'date'
+require_relative 'shifts'
 class Enigma
-  attr_reader :alphabet
+  attr_reader :alphabet, :key, :date
 
   def initialize
     @alphabet = Array("a".."z")<< " "
+    @key = rand(99999).to_s
+    @date = Date.today.strftime("%d%m%y")
+  end
+
+  def encrypt(string, key = @key, date = @date)
+    shifts = Shifts.new(key, date)
+    encryption_output(cipher(string, shifts.shifts), key, date)
+  end
+
+  def decrypt(string, key, date = @date)
+    shifts = Shifts.new(key, date)
+    decryption_output(decipher(string, shifts.shifts), key, date)
```

Comment:
Added an instance of `Shifts` to both `encrypt` and `decrypt` to allow `Shifts` to do all the functionality of turning the `key` and `date` into and then return the result to the `Enigma` class for encrypting and decrypting.